### ISSUE TITLE
[Feature Request] Support for automatic Domain Prefixing on User Name when no domain is entered on username

### ISSUE THREAD

**Issue by feren** [sddm/sddm#982]
Basically, implement Domain Login like how it works in Windows 7, for the username input on username & password textbox only themes...

Basically, add an option in configs so that if the user doesn't specify a domain in the login screen's username textbox, then upon logging in, automatically prefix '`SETDOMAINNAMEHERE\'` to the user name, and then log in with that prefixed username and their inputted password, saving the user having to put in the domain themselves, and if they do put a Domain Name, then don't prefix the domain name at all to it.

**Comment by Vogtinator** [comment]
Normally you should be able to specify that in PAM. For samba for instance using `winbind use default domain = yes` in `smb.conf`

**Comment by feren** [comment]
@Vogtinator Would that also mean the same thing happens in SDDM's user logon itself?

**Comment by Vogtinator** [comment]
Yes. Login to sddm is equivalent to login on the console. If not, it's either a bug in the PAM configuration or (less likely) in sddm itself.

**Comment by rsbrux** [comment]
I would like a login screen with a separate field for domain, instead of having to include it in the username field.
